src/main/java/me/kixstar/kixbungeebridge/Config.java

```diff
@@ -0,0 +1,46 @@
+package me.kixstar.kixbungeebridge;
+
+import net.md_5.bungee.api.plugin.Plugin;
+import net.md_5.bungee.config.Configuration;
+import net.md_5.bungee.config.ConfigurationProvider;
+import net.md_5.bungee.config.YamlConfiguration;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Config {
+
+    private static Configuration config;
+
+    private static File getDataFolder() {
+        Plugin plugin = KixBungeeBridge.getInstance();
+        if(plugin != null) return plugin.getDataFolder();
+        throw new RuntimeException("Cannot get DataFolder of plugin that isn't enabled");
+    }
+
+    private static Configuration getConfig() {
+        if(config instanceof Configuration) return config;
+        try {
+            config = ConfigurationProvider
```

Comment:
JavaPlugin have a config field already that allows you to save default config file.

Reply:
https://github.com/kixstar-mc/KixstarDungeons/blob/develop/src/main/java/me/kixstar/dungeons/KixstarDungeons.java#L43

You can see it in action there.